cicd/docupdate.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/env bash
+set -e
+
+# Update Progress page in documentation
+git clone git@github.com:ZOSOpenTools/meta.git meta_update
+cd meta_update
+python3 tools/getbinaries.py
+
+# Generate zopen API Reference
+mkdir -p docs/api
+mkdir -p docs/reference
+. ./.env
+export ZOPEN_ROOTFS="na" # To workaround sourcing zopen-config error
+mkdir -p "man/man1/"
+zopen-help2man "man/man1/" # Generate man pages
+
+cat <<EOF > docs/reference/zopen-reference.md
+# zopen reference documentation
+This page provides information about the zopen interface. Click on any of the zopen commands listed below to access the reference guide describing how to utilize that command.
+EOF
+
+# Generate html and markdown pages
```

Comment:
@IgorTodorovskiIBM ... does this look OK?

Reply:
Yes, thank you!